### PR TITLE
ignore unwanted mtd field in import

### PR DESCRIPTION
**Pull request by Paurikova2**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  2  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Problem description
Ignore metadata that we don't wanna imported based on metadata field name.

### CODE REVIEW

File: src/pump/_item.py [hunk 1]
```diff
@@ -157,7 +164,8 @@ def _import_item(self, dspace, generic_item_d, item, handles, metadatas, eperson
             'lastModified': item['last_modified'],
             'withdrawn': item['withdrawn']
         }
-        i_meta = metadatas.value(items.TYPE, i_id)
+        i_meta = (metadatas.
```
Comment (vidiecan) [inline src/pump/_item.py:167]: why superfluous () ?

**Review by vidiecan — CHANGES_REQUESTED** [review]

**Comment by Paurikova2** [comment]
@vidiecan @milanmajchrak  After the merge we need to test it on dev-5

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -6,8 +6,8 @@ repos:
     - id: mixed-line-ending
       args: ['--fix=lf']
       description: Forces to replace line ending by the UNIX 'lf' character.
--   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: v2.0.2
+-   repo: https://github.com/hhatto/autopep8
```
Comment (vidiecan) [inline .pre-commit-config.yaml:9]: I do not like this, either put description here or into the issue

File: src/pump/_metadata.py [hunk 3]
```diff
@@ -536,6 +536,12 @@ def _get_key_v2(self, val):
             key += '.' + field_js['qualifier']
         return key
 
+    def filter_res_d(self, res_d, ignored_mtd_fields):
+        """
+            Filter the resulting res_d dictionary based on custom ignored metadata fields.
+        """
+        return {key: val for key, val in res_d.items() if res_d is not None and key not in ignored_mtd_fields}
```
Comment (vidiecan) [inline src/pump/_metadata.py:543]:
```suggestion
        return {key: val for key, val in (res_d or {}).items() if key not in ignored_mtd_fields}
```

**Review by vidiecan — CHANGES_REQUESTED** [review]
?

File: .pre-commit-config.yaml (hunk 2 above)
Comment (Paurikova2) [reply .pre-commit-config.yaml:9]: In the morning, I had a problem with pre-commit. When I opened the link https://github.com/pre-commit/mirrors-autopep8, I found information about an unsupported repository and the deprecation of the new one. I also added this information to the issue

**Pull request merged**